modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +371,149 @@ ID HOST PORT RACK
 
 endif::[]
 
+== Multi-AZ deployments
+
+A multi-AZ deployment is one in which we deploy a single Redpanda cluster that has brokers distributed over multiple availability zones. The basis for this architecture is Redpanda’s rack awareness feature, in which each zone becomes a “rack”. This allows Redpanda to place replicas with respect to failure domains, resulting in a cluster that can survive a zone outage.
+
+NOTE: Adding a zone does not necessarily increase availability - it’s the replication factor of a given partition that matters. If all of your partitions use a replication factor of three, then adding an additional broker in a fourth zone just means fewer partitions are affected by an outage (since the workload is spread out more).
+
+The primary reason to deploy across multiple availability zones is to achieve extremely high availability, even at the expense of other considerations. Choosing this approach requires careful consideration of your system's requirements. Some of the considerations of a multi-AZ apprhach include:
+
+* Cost: Maintaining presence across multiple availability zones may incur additional costs. You may require additional nodes to hit the minimum requirements for utilizing a multi-AZ deployment. Data sent between availability zones is often chargeable, resulting in additional cloud costs.
```

Comment:
Node should be broker, right? See [glossary](https://deploy-preview-328--redpanda-docs-preview.netlify.app/current/reference/glossary/#node).